docs/it/user/articles.rst

```diff
@@ -45,7 +45,7 @@ Usando la vostra applicazione per smartphone
 Android
 """""""
 
-Potete scaricare `qui l'applicazione per Android <https://play.google.com/store/apps/details?id=fr.gaulupeau.apps.InThePoche>`_.
+Potete scaricare `qui l'applicazione per Android <https://play.google.com/store/apps/details?id=fr.gaulupeau.apps.InThePoche>`_ o via `F-Droid <https://f-droid.org/repository/browse/?fdid=fr.gaulupeau.apps.InThePoche>`_.
```

Comment:
"tramite" is more appropriate than "via" in this context. (sorry for the double comment).